### ISSUE TITLE
Pipe visualizer stuck rendering

### ISSUE THREAD

**Issue by ArchEzekiel** [raiguard/PipeVisualizer#31]
### Description

I pressed the hotkey to disable pipe-visualizer during an autosave, and it became stuck while disabled. re-enabling and disabling it didn't help, deleting the entities also didn't help, rendering another entity didn't clear it, and reloading the game didn't fix it
![image](https://github.com/raiguard/PipeVisualizer/assets/66336623/a69b78be-e18d-4575-9365-4fb313afe7af)

### Reproduction

1. Have hover mode enabled
2. Disable it while hovering over entities as an autosave happens

**Comment by raiguard** [comment]
I can't reproduce this, can you provide a save file and exact steps?

**Comment by riemervdzee** [comment]
I think I have the same problem in a Space Exploration save. I'm unsure how it was caused as I noticed it only after a while. Removing the buildings, or restarting the game doesn't help

![20240413183510_1](https://github.com/raiguard/PipeVisualizer/assets/510299/6078b8dd-5b41-4901-8505-118521a7b0f0)

![20240413183555_1](https://github.com/raiguard/PipeVisualizer/assets/510299/93fd5ad1-b39f-46ce-9927-75747d267430)

Save game: https://we.tl/t-ZPp8LT0CTi (the link is only valid for 7 days, as it's too big for github to upload)

If you need more information or if I should create a new issue instead please let me know

**Comment by raiguard** [comment]
@riemervdzee Sorry, could you upload your save file again? I didn't download it in time.

**Comment by riemervdzee** [comment]
@raiguard Sure thing! this link should be accessible for a longer time

https://pub-aa2c74fa42ec41f4ac28f0d50307e402.r2.dev/Riemero%20Bug%20Report.zip
